camera/src/web.ts

```diff
@@ -47,6 +47,7 @@ export class CameraWeb extends WebPlugin implements CameraPlugin {
   ) {
     if (customElements.get('pwa-camera-modal')) {
       const cameraModal: any = document.createElement('pwa-camera-modal');
+      cameraModal.facingMode = options.direction === CameraDirection.Rear ? 'environment' : 'user';
```

Comment:
Since `CameraDirection.Rear` is the default, the conditional should check for `CameraDirection.Front`, otherwise if `options.direction` is undefined it will use front camera as undefined is different from `CameraDirection.Rear`

```suggestion
      cameraModal.facingMode = options.direction === CameraDirection.Front ? 'user' : 'environment';
```

Reply:
![image](https://user-images.githubusercontent.com/12752959/146271620-5362b9fc-713f-462f-bedd-039a2d036c61.png)
For possible cases it delivers the expected result.
[https://stackoverflow.com/a/52812734/7638125](https://stackoverflow.com/a/52812734/7638125)

Reply:
Only this change has to be tested with this
[https://github.com/ionic-team/pwa-elements/pull/93](https://github.com/ionic-team/pwa-elements/pull/93)

Reply:
If no direction is passed it should use rear (environment) because it’s the default value, your image shows that if no direction is provided (undefined) it uses front camera (user)

Reply:
![image](https://user-images.githubusercontent.com/12752959/146274479-12a89422-55b0-4632-b1de-adf10e9b4c0a.png)
So with this cameraModal.facingMode = options.direction === CameraDirection.Front ? 'user' : 'environment';
Fail in 
![image](https://user-images.githubusercontent.com/12752959/146274850-4de0ecd9-a39b-4a93-bfd2-facc07059f69.png)

Reply:
Check the proposed change, it’s 
`cameraModal.facingMode = options.direction === CameraDirection.Front ? 'user' : 'environment';`

Reply:
![image](https://user-images.githubusercontent.com/12752959/146275492-b8e31c46-81bc-491c-9862-3a3831010e99.png)
ok, if I have to make another adjustment

Reply:
![image](https://user-images.githubusercontent.com/12752959/146276091-734d5962-4f08-4725-872f-0f9830aa894f.png)
and like this
![Uploading image.png…]()

Reply:
![image](https://user-images.githubusercontent.com/12752959/146276579-a52f435d-81eb-4879-9262-f500b460e709.png)

Reply:
![image](https://user-images.githubusercontent.com/12752959/146277850-34e6a3c2-361a-4b49-8abd-aa1f89e4e5fa.png)
![image](https://user-images.githubusercontent.com/12752959/146277904-6a99db4f-59d6-460b-9248-edb374588657.png)
now yes

Reply:
While the last change works as expected, I see unnecessary to create two objects for a simple value check and that approach doesn't align with the style used in the other plugins.
The change I requested was just reversing the order of your initial one line check so environment was the default.
Can you do that?

Reply:
yes

Reply:
verification remains like this:
cameraModal.facingMode = options.direction === CameraDirection.Front ? 'user' : 'environment';